runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler.cpp

```diff
@@ -216,6 +225,69 @@ OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler_initCifNativeTh
 	ffiTypeHelpers.freeStructFFIType(returnType);
 	goto done;
 }
+
+#if JAVA_SPEC_VERSION >= 22
+/* openj9.internal.foreign.abi.InternalDowncallHandler: private static native boolean isFfiProtoEnabled(); */
+VM_BytecodeAction
+OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler_isFfiProtoEnabled(J9VMThread *currentThread, J9Method *method)
+{
+	VM_BytecodeAction rc = EXECUTE_BYTECODE;
+	J9JavaVM *vm = currentThread->javaVM;
+	bool isFfiProtoOn = (J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_FFI_PROTO)) ? true : false;
```

Comment:
The return value should be of type `jboolean`:
```c
	jboolean isFfiProtoOn = J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_FFI_PROTO) ? JNI_TRUE : JNI_FALSE;
```

Reply:
Agreed and fixed given we are saving the return value back on the java stack.